.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
please remove from this file (and add `.idea` to your global `gitignore` )
> touch a `~/.gitignore` file and add the `.idea` dir to it

Reply:
I disagree, I think it's a reasonable quality of life improvement in an open source repository to ignore some *very* common IDE files.

Reply:
@darscan I was told to do this before, thus did the same :) 
👍

Reply:
I probably share @darscan , but can see the merits for an ecosystem with a lot of different tools likely used by devs.